src/rviz/default_plugin/effort_display.h

```diff
@@ -574,6 +575,15 @@ class MessageFilterJointStateDisplay: public _RosTopicDisplay
       context_->queueRender();
     }
 
+  virtual void updateQueueSize()
+    {
+      tf_filter_->setQueueSize( (uint32_t) queue_size_property_->getInt() );
```

Comment:
please use [C++ style casts](https://stackoverflow.com/a/1609185/671658) for new code